From 84ceaac19b17888059f063acc1934237c9d4727f Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <christophe@krop.fr>
Date: Thu, 24 Mar 2022 16:21:21 +0100
Subject: [PATCH] CMake: Don't hardcode the library directory name

Using INSTALL_LIBDIR is the only reliable way to get the library install directory.

Change-Id: Ib8c4fb8b4d339c63209393d7fdb3d1c3425b03a4
Amends: d1c56073b4c
Pick-to: 6.2 6.3
---
 cmake/QtBuildInternals/QtBuildInternalsConfig.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/QtBuildInternals/QtBuildInternalsConfig.cmake b/cmake/QtBuildInternals/QtBuildInternalsConfig.cmake
index a515fcd43d..c1108f583b 100644
--- a/cmake/QtBuildInternals/QtBuildInternalsConfig.cmake
+++ b/cmake/QtBuildInternals/QtBuildInternalsConfig.cmake
@@ -806,7 +806,7 @@ macro(qt_examples_build_begin)
     if(NOT QT_IS_EXTERNAL_EXAMPLES_BUILD OR NOT __qt_all_examples_ported_to_external_projects)
         qt_internal_set_up_build_dir_package_paths()
         list(APPEND CMAKE_FIND_ROOT_PATH "${QT_BUILD_DIR}")
-        list(APPEND QT_EXAMPLES_CMAKE_PREFIX_PATH "${QT_BUILD_DIR}/lib/cmake")
+        list(APPEND QT_EXAMPLES_CMAKE_PREFIX_PATH "${QT_BUILD_DIR}/${INSTALL_LIBDIR}/cmake")
     endif()

     # Because CMAKE_INSTALL_RPATH is empty by default in the repo project, examples need to have
--
2.35.1
